Add HMCTS Landing Page (#57)
ministryofjustice/data-and-analytics-engineering: src/content/about/structure/index.md

## Portfolio

> Our portfolio is our collection of work that is managed to achieve goals and deliver a strategy. To find out more about each portfolio team, please follow these links:

Reviewer: This feels a little clunky.

```suggestion
We manage and maintain analytical databases and tools which enable effective and impactful analysis across the MoJ. Our portfolio includes the following areas:
```

#### HM Prison and Probation Service (HMPPS)

The HMPPS domain in Data and Analytics engineering, are responsible for providing users on the Analytical Platform access to data covering Prisons and Probation data for analytical purposes.

Internal only [link](https://justiceuk.sharepoint.com/:p:/r/sites/DataEngineeringDataScience/Shared%20Documents/General/1.%20Data%20%26%20Analytics%20Engineering%20Hub/3.%20Prison,%20Probation%20%26%20EM/hmpps_posters.pptx?d=w17120a8f29f2469cbfbe85be1a7049d2&csf=1&web=1&e=jX6jC7) to Show and Tell posters.

#### HM Courts and Tribunals Service (HMCTS)

> The HMCTS domain in Data and Analytics engineering, are responsible for providing users on the Analytical Platform access to data covering Civil, Family, Tribunals, Magistrates, and Crown court data for analytical purposes.

Reviewer: Let's reduce the syntactic duplication.

```suggestion
Similarly to the HMPPS, the HMCTS domain in Data and Analytics engineering, are responsible for providing Data Scientists, Analysts, and Statisticians on the Analytical Platform access to court data covering Civil, Family, Tribunals, Magistrates, and Crown courts.
```

#### Corporate and Programmes

> The Corporate & Programmes domain in Data and Analytics engineering, are responsible for providing users on the Analytical Platform access to data covering Finance, People and Commercial, as well as the BOLD and Data First programmes, for Analytical Purposes.

Reviewer 2: ```suggestion
The Corporate & Programmes domain in Data and Analytics engineering, are responsible for providing users on the Analytical Platform access to data covering Finance, People and Commercial, as well as the BOLD and Data First programmes, for analytical purposes.
```

Reviewer 2: realised this was capitalised inconsistently

Internal only [link](https://justiceuk.sharepoint.com/:p:/s/DMETCorporate/EWYWM-dXQtlHguKF-Dnx8jUB8MsLZXWfamZ93GLF9cdhwA?e=S7fQ7s) to Show and Tell posters.

#### Centre for Excellence (CfE)

> [Slide deck summarising CfE](https://moj-analytical-services.github.io/dmet-cfe/).

Reviewer: We should include a one liner here about what the CfE is.

```suggestion
#### Centre for Excellence (CfE)

Our mission is to drive excellence and innovation in data and analytics engineering through continuous improvement, collaboration, and knowledge sharing.

We set the technical direction, prioritising addressing immediate challenges faced by domain teams and their partners.

You can read more about us in [these summary slides](https://moj-analytical-services.github.io/dmet-cfe/).

#### Centre for Enablement

> Under construction!

Reviewer: There is a question about if we should include it if the CfE aims are not summarised.

Reviewer 2: Unlikely this will be populated until after Phil's TRA as he's technically no longer in the team. Probably worth removing for now.

Reviewer: OK, lets remove the section for now.

#### HM Courts and Tribunals Service (HMCTS)

Similarly to the HMPPS, the HMCTS domain in Data and Analytics engineering, are responsible for providing Data Scientists, Analysts, and Statisticians on the Analytical Platform access to court data covering Civil, Family, Tribunals, Magistrates, and Crown courts.

#### Corporate and Programmes

The Corporate & Programmes domain in Data and Analytics engineering, are responsible for providing users on the Analytical Platform access to data covering Finance, People and Commercial, as well as the BOLD and Data First programmes, for analytical purposes.

Internal only [link](https://justiceuk.sharepoint.com/:p:/s/DMETCorporate/EWYWM-dXQtlHguKF-Dnx8jUB8MsLZXWfamZ93GLF9cdhwA?e=S7fQ7s) to Show and Tell posters.

#### Centre for Excellence (CfE)

Our mission is to drive excellence and innovation in data and analytics engineering through continuous improvement, collaboration, and knowledge sharing.

We set the technical direction, prioritising addressing immediate challenges faced by domain teams and their partners.

You can read more about us in [these summary slides](https://moj-analytical-services.github.io/dmet-cfe/).

> #### Centre for Enablement
> 
> Under construction!

Reviewer: ```suggestion
```
